### PR TITLE
Product crud functionality added 

### PR DESCRIPTION
**Pull request by aht007**
Admin can add, update or delete Products
User can view products in a list form and click on individual product to get its detail on a separate page

### CODE REVIEW

File: inverntory_management/inverntory_management/settings.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'products.apps.ProductsConfig',
```
Comment (awais786) [inline inverntory_management/inverntory_management/settings.py]: It should be after django default imports.

File: inverntory_management/products/apps.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ProductsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```
Comment (awais786) [inline inverntory_management/products/apps.py:5]: I think you can add this into settings file.

File: inverntory_management/products/views.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# Create your views here.
+
+def index(request):
+    products = Product.objects.order_by('-created_at')
+    context = {"products":products}
+    return render(request, 'products/index.html', context)
+
+def detail(request, prod_id):
+    try:
+        product = Product.objects.get(pk=prod_id)
```
Comment (awais786) [inline inverntory_management/products/views.py:14]: https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#get-object-or-404

File: inverntory_management/products/views.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    except Product.DoesNotExist:
+        raise Http404("Product does not exist")
```
Comment (awais786) [inline inverntory_management/products/views.py]: what would happen on front-end side it you raise404 here ?

**Comment by awais786** [comment]
https://docs.djangoproject.com/en/3.2/topics/forms/
use django forms.

File: inverntory_management/products/apps.py (hunk 2 above)
Comment (aht007) [reply inverntory_management/products/apps.py:5]: This file was generated automatically by the Django boilerplate..... Should I keep it this way or add this stuff to setting file? 
Meanwhile I will look into other issues.

**Pull request merged**